app/src/main/java/space/taran/arknavigator/navigation/Screens.kt

```diff
@@ -17,6 +18,15 @@ class Screens {
         override fun getFragment() = ResourcesFragment.newInstance(rootAndFav)
     }
 
+    class ResourcesScreenWithSelectedTag(
+        val rootAndFav: RootAndFav,
+        val tag: Tag
```

Comment:
Better to do it in more generic way here with "selected query" not "selected tag". Query is a set of tags (which is the user's input). `TagsSelectorPresenter` contains `includedTags` which is, basically, selected query. It should be re-used.